tests/sentry/api/endpoints/test_project_rule_details.py

```diff
@@ -616,63 +416,33 @@ def test_update_channel_slack(self):
             body=json.dumps({"ok": channels["ok"], "channel": channels["channels"][1]}),
         )
 
-        response = self.client.put(
-            url,
-            data={
-                "name": "#new_channel_name",
-                "actionMatch": "any",
-                "filterMatch": "any",
-                "actions": actions,
-                "conditions": conditions,
-                "frequency": 30,
-            },
-            format="json",
+        payload = {
+            "name": "#new_channel_name",
+            "actionMatch": "any",
+            "filterMatch": "any",
+            "actions": actions,
+            "conditions": conditions,
+            "frequency": 30,
+        }
+        self.get_success_response(
+            self.organization.slug, self.project.slug, self.rule.id, status_code=200, **payload
         )
-
-        assert response.status_code == 200, response.content
-        rule = Rule.objects.get(id=response.data["id"])
-        assert rule.label == "#new_channel_name"
-        assert rule.data["actions"][0]["channel_id"] == "new_channel_id"
+        # TODO(Leander): Figure out a way to just run assert_from_payload)
```

Comment:
dangling paren

Reply:
I will actually look at addressing this TODO, I forgot about it when I started fixing the `test_project_rules` test file. TY

Reply:
Okay the TODO was addressed!